internal/internal_task_pollers_test.go

```diff
@@ -143,8 +130,6 @@ func TestWFTRacePrevention(t *testing.T) {
 	t.Log("Issue task1")
 	go func() { resultsChan <- poller.processWorkflowTask(&task1) }()
 
-	require.Never(t, tryWrite(completionChans[1]), 2*time.Second, 100*time.Millisecond,
-		"Should be no reader on the task1's completion channel as task0 holds the ctx lock")
```

Comment:
The way that this poor `require.Never` is written, the callback can technically be invoked even after this is returned, which was causing race conditions with the closing of the channel later. Decided this part of the test isn't really needed to test the rest.